### PR TITLE
feat: add first html lab protoype

### LINKED ISSUE #380: chore - add first basic HTML lab project for frontend cert
The basic HTML section will have a few labs for campers to do.
Labs are defined as short projects campers build on their own. It will consist of a blank editor and user stories.
Here is an example for the format:
https://www.freecodecamp.org/learn/the-odin-project/top-build-a-recipe-project/top-build-a-recipe-project

For this issue, we need a lab that covers the following basic HTML topics: 

- headings
- images
- lists
- paragraphs

## Ideas for lab projects

- Personal Profile Page
- Recipe Page
- Travel Page
- Product Showcase Page

## Note about user stories

Since this will be the first lab campers will create we should aim for 8 user stories or less. This should just be a short project for them to do. 

## Note about adding new images

For adding new images, make sure they are free to use. You will also need to be add that image to our CDN repo here
https://github.com/freeCodeCamp/cdn

Once the new image has been added to the CDN repo, then you will need to reference that cdn link in your project.

If you are not sure about if an image can be used, reach out on staff chat. 

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/html-projects` directory
- [ ] add your completed project in an `index.html` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #380

I added the image here. Once this PR is approved I will open a PR to upload the image to the cdn and fix the `src` path.

### CODE REVIEW

File: frontend-cert/html-projects/chocolate-cookies-recipe/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+1. You should have  a`title` element containing the text `Chocolate Chip Cookies Recipe` and a `meta` element with a `charset` attribute set to `UTF-8` in the `head`.
```
Comment (moT01) [inline frontend-cert/html-projects/chocolate-cookies-recipe/user-stories.md:1]:
```suggestion
1. You should have a `title` element containing the text `Chocolate Chip Cookies Recipe` and a `meta` element with a `charset` attribute set to `UTF-8` in the `head`.
```

File: frontend-cert/html-projects/chocolate-cookies-recipe/user-stories.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+1. You should have an `h1` element with the text `Chocolate Chip Cookies`.
+
+1. You should have a `p` element that introduces the recipe below the `h1`.
+
+1. You should have one `h2` element with the text `Ingredients` for the ingredients section.
+
+1. You should have an unordered list element with at least 4 list items that lists your ingredients below the first `h2` element.
+
+1. You should have a second `h2` element with the text `Method` for the method section.
```
Comment (moT01) [inline frontend-cert/html-projects/chocolate-cookies-recipe/user-stories.md:11]:
```suggestion
1. You should have a second `h2` element with the text `Instructions` for the instructions section.
```

`Method` sounds a little strange to me. We don't have to change it, but I like `Instructions` better.

**Review by moT01 — COMMENTED** [review]
I left a few minor suggestions.

File: frontend-cert/html-projects/chocolate-cookies-recipe/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        <p>lorem ipsum</p>
+        <img src="cookies-image.jpg" alt="a tray of chocolate chip cookies">
+        <h2>Ingredients</h2>
+        <ul>
+            <li>test</li>
+            <li>test</li>
+            <li>test</li>
+            <li>test</li>
+        </ul>
+        <h2>Method</h2>
```
Comment (Dario-DC) [inline frontend-cert/html-projects/chocolate-cookies-recipe/index.html:18]:
```suggestion
        <h2>Instructions</h2>
```

**Review by moT01 — APPROVED** [review]
LGTM 👍 Feel free to merge it you are satisfied @Dario-DC

**Pull request merged**